opacclient/libopac/src/main/java/de/geeksfactory/opacclient/objects/AccountItem.java

```diff
@@ -0,0 +1,148 @@
+package de.geeksfactory.opacclient.objects;
+
+public abstract class AccountItem {
+    private long account;
+    private String title;
+    private String author;
+    private String format;
```

Comment:
We could also provide both a String property and a `SearchResult.MediaType` property so we can choose what works for each system in the APIs and show the one that is available in the app (and optionally show an icon if the `SearchResult.MediaType` property is set).